packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -127,7 +127,8 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
   }
 
   public async _onSync(): Promise<TransactionReceipt> {
-    const pendingQueueElements = await this.chainContract.getNumPendingQueueElements()
+    const pendingQueueElements =
```

Comment:
same q as above ... why the lint changes? these are going to be unnecessary merge conflicts later on.

Reply:
found the tslint<>prettier conflict and fixed